app/src/main/java/app/revanced/integrations/patches/ForceDisableCaptionsPatch.java

```diff
@@ -3,8 +3,10 @@
 import app.revanced.integrations.settings.SettingsEnum;
 
 public class ForceDisableCaptionsPatch {
```

Comment:
Some changes have been requested in the PR for the [related patch](https://github.com/revanced/revanced-patches/pull/435/files). Make sure to apply them to the integrations too, such as the class name.